.yamato/upm-ci.yml

```diff
@@ -51,8 +51,8 @@ test_{{ platform.name }}_{{ editor.version }}:
         - "upm-ci~/test-results/**/*"
   dependencies:
     - .yamato/upm-ci.yml#pack
-{% endfor %}
-{% endfor %}
+      {% endfor %}
+      {% endfor %}
```

Comment:
No need to move this one, also it's better if it's stay aligned with the "begin"for.

Reply:
Oh you're right. I'll fix that.